src/sensors/FakeLocation.ts

```diff
@@ -0,0 +1,78 @@
+import { EventEmitter } from 'events';
+import { ISensor } from './Sensor';
+import * as fs from 'fs';
+
+export class FakeLocation extends EventEmitter implements ISensor {
+
+	private readonly LocationSentences: Object[] = [];
+	private sentenceIndex: number = 0;
+	private started = false;
+	private locationEmitterIntervalId: any = null;
+
+	constructor(
+		private readonly locationReading: string,
+		private readonly loop: boolean = false,
+	) {
+		super();
+	}
+
+	private readLocationData() {
+		const data = JSON.parse(fs.readFileSync(this.locationReading, 'utf8'));
+		this.LocationSentences.push(...data.locations);
+		this.cleanUpAndStartEmitting();
+	}
+
+
+	private emitGPSData() {
+
+		this.emit(
+			'data',
+			Date.now(),
+			this.LocationSentences[this.sentenceIndex]
+		);
+
+		if (this.sentenceIndex === this.LocationSentences.length - 1) {
+			if (this.loop) {
+				this.sentenceIndex = 0;
+			} else {
+				this.stop();
+			}
+		} else {
+			this.sentenceIndex++;
+		}
+	}
+
+	async start(): Promise<void> {
+		const fileExists = await new Promise(resolve =>
+			fs.exists(this.locationReading, resolve),
+		);
+
+		if (!fileExists) {
+			throw new Error(
+				`Recording with filename '${this.locationReading}' does not exist.`,
+			);
+		}
+
+		this.started = true;
+
+		this.readLocationData();
+	}
+
+	private cleanUpAndStartEmitting() {
+		if (this.LocationSentences) {
+			this.locationEmitterIntervalId = setInterval(() => {
+				this.emitGPSData();
+			}, 3000);
```

Comment:
Does this mean it will send a location every 3 seconds? 
That's probably too often, esp for the FE rendering the locations on the map. I've stopped using the `DEVICE` appId because it sends a message every 1 second, which is a relative flood that can cause problems after a while. The FE keeps shows incoming messages for a full minute after 3 minutes of the simulated device sending `DEVICE` messages, so things are backing up. Not sure where, but that's a subject for later. 

Every 10 seconds would be safer and fast enough, thoughts from anyone? Devs can edit their local code to make it more frequent when they want to.

So, unless anyone has qualms, could you also set the DEVICE default frequency to 5 seconds in this PR too?